client/src/client.rs

```diff
@@ -236,7 +234,7 @@ impl From<ResponseReport> for eyre::Report {
 }
 
 /// Output of a query
-pub trait QueryOutput: Into<Value> + TryFrom<Value> {
+pub trait QueryOutput: Into<Value> + TryFrom<Value> + Decode {
```

Comment:
```suggestion
pub trait QueryOutput: Decode {
```

I guess the same question